src/Microsoft.Sbom.Targets/Microsoft.Sbom.Targets.csproj

```diff
@@ -39,6 +41,15 @@
     </ItemGroup>
   </Target>
 
+   <Target Name="AddSbomToolToPackage" AfterTargets="Build">
```

Comment:
We should figure out if there is a better AfterTargets for SBOM generation to only do this when they need an SBOM. Can do in a follow-up change once that is figured out.

Reply:
This target is for including the SBOM CLI Tool into the Task's Nuget Package. I tried making the target run only during the packing step without success :(